xarray/core/computation.py

```diff
@@ -766,11 +767,10 @@ def func(*arrays):
         not isinstance(result_data, tuple) or len(result_data) != signature.num_outputs
     ):
         raise ValueError(
-            "applied function does not have the number of "
-            "outputs specified in the ufunc signature. "
-            "Result is not a tuple of {} elements: {!r}".format(
-                signature.num_outputs, result_data
-            )
+            f"applied function does not have the number of "
+            f"outputs specified in the ufunc signature. "
+            f"Result is not a tuple of {signature.num_outputs} elements:\n\n"
```

Comment:
Does this work well if we receive a list of arrays?

Perhaps we should add 
```python
f"Received a {type(result_data)} with {len(result_data)} elements instead."
```

Reply:
Ah, good point. #8192